app/models/organisation.rb

```diff
@@ -1,6 +1,6 @@
 class Organisation < ApplicationRecord
   has_one_attached :signed_mou
-  has_many :users, inverse_of: :organisation, dependent: :destroy
+  has_and_belongs_to_many :users, inverse_of: :organisation
```

Comment:
This means we might end up with users with no orgs. Not an issue to solve now though I don't think